modules/percolator/src/main/java/org/elasticsearch/percolator/QueryAnalyzer.java

```diff
@@ -341,9 +348,29 @@ public static Result analyze(Query query) {
         };
     }
 
+    static Function<Query, Result> pointRangeQuery() {
+        return query -> {
+            PointRangeQuery pointRangeQuery = (PointRangeQuery) query;
+            byte[] lowerPoint = pointRangeQuery.getLowerPoint();
+            byte[] upperPoint = pointRangeQuery.getUpperPoint();
+            byte[] interval = new byte[16];
+            NumericUtils.subtract(pointRangeQuery.getBytesPerDim(), 0, prepad(upperPoint), prepad(lowerPoint), interval);
```

Comment:
Right, it is more consistent to use 16 instead. Although I think it shouldn't cause issues as the prepadded bytes are always zero.